src/qt/addressbookpage.cpp

```diff
@@ -21,6 +21,8 @@
 #include <QMessageBox>
 #include <QSortFilterProxyModel>
 
+
```

Comment:
why are here these spaces?

Reply:
Nonsenee, might be an issue of search & replace. I will close this and reopen later.